app/models/like.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  # Callbacks
+  after_save :update_post_likes_counter
+
+  # Methods
+  def update_post_likes_counter
+    post.update(likes_counter: post.likes.count)
```

Comment:
- [ ] [OPTIONAL]
Again, let's follow best practice by marking `update_post_likes_counter` as _private_ after implementing the `after_save` method.